prody/proteins/header.py

```diff
@@ -876,6 +908,91 @@ def _getNumModels(lines):
         except:
             pass
 
+def _getCRYST1(lines):
+    line = lines['CRYST1']
+    if line:
+       i, line = line[0]
+       try:
+           return {'cellLength': (float(line[6:15]),
+                                  float(line[15:24]),
+                                  float(line[24:33])),
+                   'cellAngles': (float(line[33:40]),
+                                  float(line[40:47]),
+                                  float(line[47:54])),
+                   'spaceGroup': line[55:66].strip(),
+                   'Z value' : int(line[66:70])
+                   }
+       except:
+           pass
+
+def _missingResidues(lines):
+    """
+    Parse REMARK 465, Missing residues records
+    REMARK 465   M RES C SSSEQI
+    REMARK 465     GLU B   448
+    header['missing_residues'] = [(chid, resname, resnum, icode, modelNum)]
+    """
+    mr = []
+    header = True
+    for i, line in lines['REMARK 465']:
+        #skip header records
+        if line.startswith("REMARK 465   M RES C SSSEQI"):
+            header=False
+            continue
+        if header: continue
+        modelNumStr = line[10:14].strip()
+        if modelNumStr: modelNum = int(modelNumStr)
+        else: modelNum = None
+        w = line[15:].split()
+        icode = ''
+        if w[2][-1].isalpha():
+            icode = w[2][-1]
+            resnum = int(w[2][:-1])
+        else:
+            resnum = int(w[2])
+        mr.append((w[1],w[0],resnum, icode, modelNumStr))
+    return mr
+    
+def _missingAtoms(lines):
+    """
+    Parse REMARK 470,  Missing Atom records
+    REMARK 470   M RES CSSEQI  ATOMS
+    REMARK 470     ARG A 412    CG   CD   NE   CZ   NH1  NH2  
+    header['missing_atoms'] = [(chid, resname, resnum, icode, modelNum, [atom names])]
+    """
+    ma = []
+    seen = {}
+    header = True
+    for i, line in lines['REMARK 470']:
+        #skip header records
+        if line.startswith("REMARK 470   M RES CSSEQI  ATOMS"):
+            header=False
+            continue
+        if header: continue
+        modelNumStr = line[10:14].strip()
+        if modelNumStr: modelNum = int(modelNumStr)
+        else: modelNum = None
+        #w = line[15:].split()
```

Comment:
remove commented code or explain why it's here

Reply:
I removed the code